IntroOOP/RationalNumbers/RatNum.cs

```diff
@@ -0,0 +1,369 @@
+﻿namespace IntroOOP.RationalNumbers;
+
+
+public class RatNum
+{
+    private int _Numerator;     // Числитель
+    private int _Denominator;   // Знаменатель
```

Comment:
Кроме того, надо стараться делать объекты неизменяемыми. Это избавит Вас в дальнейшем от многих граблей.
```suggestion
    private readonly int _Numerator;     // Числитель
    private readonly int _Denominator;   // Знаменатель
```
Любая операция - это новый объект с изменённым состоянием. Но это касается не всех возможных объектов в вашей программе...